### PR TITLE
Setup GH Actions

### PR DESCRIPTION
**Pull request by jpadilla**
Heavily based on the awesome setup from https://github.com/python-attrs/attrs.
This will remove the need for appveyor.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,40 +1,88 @@
...
+  tests:
+    name: "Python ${{ matrix.python-version }} on ${{ matrix.platform }}"
+    runs-on: "${{ matrix.platform }}"
+    env:
+      USING_COVERAGE: '3.7,3.8'
+
+    strategy:
+      matrix:
+        platform: ["ubuntu-latest", "windows-latest"]
+        python-version: ["3.5", "3.6", "3.7", "3.8"]
```
Comment (auvipy) [inline .github/workflows/main.yml:23]: as python 3.5 will be EOL in October 2020, we can drop it now

File: setup.cfg [hunk 2]
```diff
@@ -37,11 +37,24 @@ console_scripts =
...
-    pytest>=4.0.1,<5.0.0
-    pytest-cov>=2.6.0,<3.0.0
+    pytest>=4.3.0,<5.0.0
+    coverage[toml]>=5.0.2
+dev =
+    sphinx
+    sphinx-rtd-theme
+    zope.interface
+    cryptography>=1.4
+    pytest>=4.3.0,<5.0.0
```
Comment (auvipy) [inline setup.cfg:54]: pytest 6 is latest

File: docs/conf.py [hunk 3]
```diff
@@ -1,38 +1,46 @@
...
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
-
+# -*- coding: utf-8 -*-
```
Comment (auvipy) [inline docs/conf.py:1]: isn;t python 2 support dropped?

File: tox.ini [hunk 4]
```diff
@@ -1,27 +1,94 @@
...
+addopts = -ra
+testpaths = tests
+filterwarnings =
+    once::Warning
+    ignore:::pympler[.*]
+
+
+[gh-actions]
+python =
+    2.7: py27
```
Comment (auvipy) [inline tox.ini:12]: python 2.7 is EOL

File: docs/conf.py [hunk 5]
```diff
@@ -282,22 +125,10 @@ def get_version(package):
     (
         master_doc,
         "PyJWT",
-        "PyJWT Documentation",
+        u"PyJWT Documentation",
```
Comment (auvipy) [inline docs/conf.py:128]: wy u?

**Review by auvipy — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 5 above)
Comment (jpadilla) [reply docs/conf.py:127]: woops

**Review by auvipy — APPROVED** [review]

**Comment by jpadilla** [comment]
thanks @auvipy I have to cleanup badges and remove integrations next.

**Comment by auvipy** [comment]
ok great!

**Pull request merged**